packages/protocol/contracts/L1/libs/LibUtils.sol

```diff
@@ -129,6 +119,22 @@ library LibUtils {
         }
     }
 
+    function calcBlockFee(
+        TaikoData.Config memory config,
+        uint64 gasAmount,
+        uint32 feePerGas
+    )
+        internal
+        pure
+        returns (uint64)
+    {
+        uint64 _gas =
+            gasAmount + LibL2Consts.ANCHOR_GAS_COST + config.blockFeeBaseGas;
+        unchecked {
+            return _gas * feePerGas;
+        }
```

Comment:
```suggestion
        unchecked {
            return gasAmount * feePerGas;
        }
```
The gas limit is now always the block gas limit, so I think we could just do this now?

Reply:
this method will be removed